contrib/tests/templates.rs

```diff
@@ -1,17 +1,26 @@
+extern crate rocket;
 extern crate rocket_contrib;
 
 use std::env;
 use std::path::PathBuf;
 
+use rocket::Rocket;
+use rocket::config::{Config, Environment};
+use rocket_contrib::Template;
+
 fn template_root() -> PathBuf {
     let cwd = env::current_dir().expect("current working directory");
     cwd.join("tests").join("templates")
 }
 
+fn rocket() -> Rocket {
+    let config = Config::build(Environment::Development).extra("template_dir", template_root().to_str().unwrap()).unwrap();
```

Comment:
This line is far too long. Split this into multiple lines, method chaining style. Also, use `expect()` instead of `unwrap()`.